deploy/operator.yaml

```diff
@@ -65,11 +67,3 @@ spec:
         - key: "node-role.kubernetes.io/master"
           operator: "Exists"
           effect: "NoSchedule"
-        - key: "node.kubernetes.io/unreachable"
-          operator: "Exists"
-          effect: "NoExecute"
-          tolerationSeconds: 120
-        - key: "node.kubernetes.io/not-ready"
-          operator: "Exists"
-          effect: "NoExecute"
-          tolerationSeconds: 120
```

Comment:
Ok - so by removing these here, we're inheriting a set of global tolerances somewhere?

Reply:
was this here to prevent pods from getting evicted on node updates? @JAORMX